src/fetch.py

```diff
@@ -284,4 +387,4 @@ def fetch(self):
 
 
 if __name__ == "__main__":
-    Fetch("https://localhost:9999/pf-admin-api/v1/api-docs").fetch()
+    Fetch("https://localhost:9999/pf-admin-api/v1/swagger.json", swagger_version='2.0').fetch()
```

Comment:
I am not sure it is a good idea to hard code this here. Can it be extracted to an env file or similar?